bolt/lib/Profile/StaleProfileMatching.cpp

```diff
@@ -360,25 +371,37 @@ createFlowFunction(const BinaryFunction::BasicBlockOrderType &BlockOrder) {
   // Add dummy edges to the extra sources. If there are multiple entry blocks,
   // add an unlikely edge from 0 to the subsequent ones
   assert(InDegree[0] == 0 && "dummy entry blocks shouldn't have predecessors");
-  for (uint64_t I = 1; I < Func.Blocks.size(); I++) {
```

Comment:
Keep Func.Blocks.size() and add `-1` to skip SinkBlock. Add a comment about skipping it.